modules/support-reviewing-an-access-request-from-an-email-notification.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-support.adoc
+// * support/getting-support.adoc
+// * service_mesh/v2x/ossm-troubleshooting-istio.adoc
+// * osd_architecture/osd-support.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="support-reviewing-an-access-request-from-an-email-notification_{context}"]
+= Reviewing an access request from an email notification
+
+.Prerequisites
+
+ifndef::openshift-rosa,openshift-dedicated[]
+* You have access to the cluster as a user with the `cluster-admin` role.
+endif::openshift-rosa,openshift-dedicated[]
+
+
+.Procedure
+
+. Click the link within the email to bring you to the {hybrid-console-second}.
+
+. Click *Approve* or *Deny* under *Decision* in the *Access Request Details* modal.
++
+[NOTE]
+====
+Denying an access request will require you to complete the *Justification* field and will require further support case engagement, to help troubleshoot the issue, given that a Red{nbsp}Hat SRE would not be able to directly act on the resources related to the support request.
+====
+
+. Click *Save*.
```

Comment:
@Ginxo Can you confirm the wording here. Is it `Save` on the UI to submit a Decision for an Access Request?

Reply:
Confirmed with Enrique it is `Save`